src/components/NurseryWithdrawals/PlantingProgressList.tsx

```diff
@@ -73,19 +74,24 @@ export type PlantingProgressListProps = {
 };
 
 export default function PlantingProgressList({ search, plantingCompleted }: PlantingProgressListProps): JSX.Element {
+  const [hasZones, setHasZones] = useState<boolean | undefined>();
   const classes = useStyles();
-  const data = useAppSelector((state: any) => searchPlantingProgress(state, search, plantingCompleted));
+  const data = useAppSelector((state: any) => searchPlantingProgress(state, search.trim(), plantingCompleted));
 
-  const hasZones = data?.some((d) => d.subzoneName);
+  useEffect(() => {
+    if (data && hasZones === undefined) {
+      setHasZones(data.some((d) => d.subzoneName));
+    }
+  }, [data, hasZones]);
 
-  if (!data) {
+  if (!data || hasZones === undefined) {
     return <CircularProgress sx={{ margin: 'auto' }} />;
   }
 
   return (
     <Box>
       <Table
-        id='plantings-progress-table'
+        id={`plantings-progress-table-${hasZones}`}
```

Comment:
Would this id be planting-progress-table-true and planting-progress-table-false? Why do we need this id?

Reply:
We use this id to store user preferences for the table column ordering. I can rename it to with/without-zones

Reply:
See https://github.com/terraware/terraware-web/blob/main/src/components/common/table/index.tsx#L54

Reply:
Thank you!!